apps/np-atobarai/src/__tests__/mocks/atobarai/mocked-atobarai-customer.ts

```diff
@@ -0,0 +1,3 @@
+import { AtobaraiCustomer } from "@/modules/atobarai/atobarai-customer";
+
+export const mockedAtobaraiCustomer = new AtobaraiCustomer();
```

Comment:
question: I don't quite get why do we need this? If we're using the real thing why create file for mocking? 🤔